index.html

```diff
@@ -210,18 +373,21 @@ <h2 class="frame">Frameworks</h2>
       <h2 class="skill">Skills</h2>
       <i  class="fa">&#xf105;</i>
     </li>
-    
-
   </ul>
+</div>
 </section>
 <section class="contact-form">
```

Comment:
- [ ] Please I highly recommend sticking to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-848&t=Hq64bUbN7WFDPuHZ-0). 😄 

_**REQUIRED FIGMA DESIGN**_

![Screenshot (204)](https://user-images.githubusercontent.com/78104544/230513072-62ace851-8d8b-4f38-bb3d-28e085ca9981.png)


_**YOUR DESIGN**_

- [ ] Please I highly recommend adding some margins to the content on the contact form and also please check the positioning of the designs to fit the required figma design. 😄 

![Screenshot (205)](https://user-images.githubusercontent.com/78104544/230513125-96b928f2-19f6-4dce-b6eb-bb82fb40c50c.png)